### ISSUE TITLE
Config options for logs

### ISSUE THREAD

**Issue by JasmineMassii** [Itokoyamato/TokoVOIP_TS3#134]
After installing 1.5.1 (full FXServer install), I noticed a lot of [LOG] statements being printed to the console.
Personally, I'm the type of person who prefers not to see data outside of the FiveM server doing its thing, along with any problems. But I recognise the value to some for those logs.

I would like to suggest a config option be added which would allow the user to choose either...
- true|false (write log to console, or do not write log to console); or
- none|console|file, or 0|1|2 for same options (no logs written or saved | logs printed to console only | logs saved to filename of users choice).
Or, any other options that might be thought out.

Errors, of course, should always be printed to console, no matter user choice. Could include them in-file if user chooses that option.

**Comment by Itokoyamato** [comment]
Agreed, it spams a lot. Will have an option for it, with logs disabled by default
